portfolio/src/test/java/com/google/sps/DataServletTest.java

```diff
@@ -64,151 +77,120 @@ public void tearDown() {
     helper.tearDown();
   }
 
-  private String[] convertStringToArray(String responseOutput) {
-      //Gets rid of brackets
+  private List<String> convertStringToList(String responseOutput) {
+      // Gets rid of brackets
       responseOutput = responseOutput.replace("[", "")
                                      .replace("]", "")
                                      .replace("\"", "");
-      //Parse by commas
-      String[] comments = (responseOutput.trim()).split(",");
-      return comments;
+      // Parse by commas
+      List<String> commentsList = new ArrayList<>();
+      String[] commentsArray = (responseOutput.trim()).split(",");
+      for(String comment : commentsArray) {
+          commentsList.add(comment);
+      }
+      return commentsList;
+  }
+  
+  private int getNumberOfEntiresInDatastore() {
+      return ds.prepare(new Query(dataServlet.COMMENT_TABLE_NAME)).countEntities(withLimit(10));
+  }
+
+  private void addEntityToDatastore(String comment_text, long timestamp) {
+    Entity comment = new Entity(dataServlet.COMMENT_TABLE_NAME);
+    comment.setProperty(dataServlet.COMMENT_COLUMN_NAME, comment_text);
+    comment.setProperty(dataServlet.TIMESTAMP_COLUMN_NAME, timestamp);
+    ds.put(comment);
+  }
+
+  private void verifyReturnedComments(StringWriter writer, String[] expectedComments) {
```

Comment:
Consider using [varargs](https://docs.oracle.com/javase/8/docs/technotes/guides/language/varargs.html) for `expectedComments` - i think it will make your tests more readable because you won't need to build arrays.

Reply:
done!